canonical/ubuntu-advantage-client
# realtime kernel

I think we're just waiting on these things:

* [ ] realtime-kernel metapackage - not a blocker for this PR, as it is configured on the backend and would only require test changes in our codebase
* [x] packages in production esm.ubuntu.com
* [x] updated contract definition

## Proposed Commit Message


```
realtime: new entitlement realtime-kernel

Includes new entitlement class and tests to cover enabling the new
service.
```

## Test Steps

Run the new integration tests in features/realtime-kernel.feature

Note that they only pass on staging.

Make sure having this new entitlement in place doesn't break anything when working with production.

## Desired commit type


## Checklist
